apps/atomy-q/API/app/Http/Controllers/Api/V1/Concerns/InteractsWithAiAvailability.php

```
@@ -37,27 +47,109 @@ protected function aiCapabilityAvailable(string $featureKey): bool
-                    'mode' => $statusSnapshot->mode,
-                ],
+                'reason_codes' => $resolvedReasonCodes,
+                'diagnostics' => $resolvedDiagnostics,
             ],
         ], $statusCode);
     }
+
+    private function fallbackAiStatusSnapshot(): AiStatusSnapshot
+    {
+        $mode = config('atomy.ai.mode');
+        $normalizedMode = is_string($mode) && in_array($mode, AiStatusSchema::modes(), true)
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\Nexus\IntelligenceOperations\DTOs\AiStatusSchema' in method 'fallbackAiStatusSnapshot'. (undefined)

(StaticAccess)